coreclient/src/conversations/messages/mod.rs

```diff
@@ -41,17 +42,25 @@ impl TimestampedMessage {
         application_message: ApplicationMessage,
         ds_timestamp: TimeStamp,
         sender_name: QualifiedUserName,
-    ) -> Result<Self, tls_codec::Error> {
-        let content = MimiContent::tls_deserialize_exact_bytes(&application_message.into_bytes())?;
```

Comment:
Previously, a single bad message caused the entire handling process to fail.

Reply:
Right, we should be more tolerant here. If/How we display invalid messages is a product question that we need to answer at some point. Just displaying an error should be good enough.